### ISSUE TITLE
org.elasticsearch.index.translog.TranslogTests#testStats failing on 6.8

### ISSUE THREAD

**Issue by dliappis** [elastic/elasticsearch#55064]
Observed a failure on branch 6.8 for `TranslogTests` in `#testStats`:

```
  | FAILURE 0.02s J15 \| TranslogTests.testStats <<< FAILURES! |  
  | > Throwable #1: java.lang.AssertionError: |  
  | > Expected: a value greater than <1L> |  
  | >      but: <1L> was equal to <1L> |  
  | > 	at __randomizedtesting.SeedInfo.seed([6B2A44E9E0939A49:F1AB1D56884F546A]:0) |  
  | > 	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20) |  
  | > 	at org.elasticsearch.index.translog.TranslogTests.testStats(TranslogTests.java:439) |  
  | > 	at java.lang.Thread.run(Thread.java:748)
```
Reproduction line:

`./gradlew ':server:unitTest' -Dtests.seed=6B2A44E9E0939A49 -Dtests.class=org.elasticsearch.index.translog.TranslogTests -Dtests.method="testStats" -Dtests.security.manager=true -Dtests.locale=ru -Dtests.timezone=SystemV/MST7MDT -Dcompiler.java=12 -Druntime.java=8`

Unfortunately I wasn't able to reproduce after 50 iterations on the same operating system.

CI link: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.8+matrix-java-periodic/ES_RUNTIME_JAVA=adoptopenjdk8,nodes=general-purpose/622/console / https://gradle-enterprise.elastic.co/s/6xn7qjvruu5x6

Interestingly enough this is the first time if fails on `6.8`, however, there were a number of failures on `master` in February that got fixed by #51905.

@dnhatn :

The specific line in 6.8 asserts >1 in https://github.com/elastic/elasticsearch/blob/fe9394ca81417f02ea5bd3e9ed02029cf394ba58/server/src/test/java/org/elasticsearch/index/translog/TranslogTests.java#L439

however, in #51905, we started asserting >0 https://github.com/elastic/elasticsearch/blob/6d28596ead7f95182b8bd112c3a314d540728126/server/src/test/java/org/elasticsearch/index/translog/TranslogTests.java#L420

Is there a reason for this difference?

**Comment by dnhatn** [comment]
Thanks @dliappis. I've opened https://github.com/elastic/elasticsearch/pull/55085.

**Comment by dnhatn** [comment]
Fixed in #55085
